setup.py

```diff
@@ -151,10 +150,6 @@
                 "share/snapcraft/libraries",
                 ["libraries/" + x for x in os.listdir("libraries")],
             ),
-            (
-                "share/snapcraft/extensions",
-                ["extensions/" + x for x in os.listdir("extensions")],
```

Comment:
Wait, no, we need these or templates won't have any parts. I'm not clear what issue you're trying to solve?

Reply:
I was overeager

Reply:
hah, extensions are at the top, hmm; does not break anything now, but sure, let's keep this

Reply:
```
(dist) ubuntu@snapcraft-xenial-dev:~/source/snapcraft/dist$ pip install snapcraft-2.43.1.tar.gz 
Processing ./snapcraft-2.43.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-i_fslbgw-build/setup.py", line 156, in <module>
        ["extensions/" + x for x in os.listdir("extensions")],
    FileNotFoundError: [Errno 2] No such file or directory: 'extensions'
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-i_fslbgw-build/
You are using pip version 8.1.1, however version 18.0 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```

But I think I can re-purpose the PR as I found the actual culprit

Reply:
oh, for this to manifest, you need to be on a clean venv